cypress/page/shipping-step.page.ts

```diff
@@ -3,11 +3,11 @@ class ShippingStepPage {
     private termsCheck: string;
 
     constructor() {
-        this.checkbtn = ".cart_navigation > .button > span";
+        this.checkbtn = ".cart_navigation .button";
```

Comment:
review if you can use the title attribute to indicate you are referring to the proceedToCheckout button

Reply:
All done @holgiosalos